modules/installation-configuration-parameters.adoc

```diff
@@ -1007,6 +1007,12 @@ For a private cluster, specify a private subnet for each availability zone.
 For clusters that use AWS Local Zones, you must add AWS Local Zone subnets to this list to ensure edge machine pool creation.
 |Valid subnet IDs.
 
+|platform:
+  aws:
+    publicIpv4Pool:
+|The public IPv4 pool ID that is used to allocate Elastic IPs (EIPs) when `publish` is set to `External`. You must provision and advertise the pool in the same AWS account and region of the cluster. You must have 2n + 1 IPv4 available in the pool where n is the total number of AWS zones used to deploy the Network Load Balancer (NLB) for API, NAT gateways, and bootstrap node. For more information about Bring your own IP addresses (BYOIP) in AWS, see link:https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-byoip.html#byoip-onboard[Onboard your BYOIP]
```

Comment:
```suggestion
|The public IPv4 pool ID that is used to allocate Elastic IPs (EIPs) when `publish` is set to `External`. You must provision and advertise the pool in the same AWS account and region of the cluster. You must have 2n + 1 IPv4 available in the pool where n is the total number of AWS zones used to deploy the Network Load Balancer (NLB) for API, NAT gateways, and bootstrap node. For more information about Bring your own Public IP addresses (BYO Public IPv4) in AWS, see link:https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-byoip.html#byoip-onboard[Onboard your BYOIP]
```

@shubham-pampattiwar we are also discussion [here][slack] to add additional permissions here: https://docs.openshift.com/container-platform/4.15/installing/installing_aws/installing-aws-account.html#installation-aws-permissions_installing-aws-account

A new section must be added when the feature is used, it is checked in [permissions.go]:
- https://github.com/openshift/installer/blob/master/pkg/asset/installconfig/aws/permissions.go#L47C2-L47C26
- https://github.com/openshift/installer/blob/master/pkg/asset/installconfig/aws/permissions.go#L269-L274

@r4f4 any additional thoughts here?

cc @bscott-rh

Reply:
I think that's it: mentioning in the docs that 2 new permissions are required when enabling the feature.